README.markdown

```diff
@@ -178,7 +176,7 @@ Disables the monitoring facility in NTP. Valid options: 'true' or 'false'. Defau
 
 Specifies an NTP driftfile. Valid options: string containing an absolute path. Default value: '/var/lib/ntp/drift' (except on AIX and Solaris)
 
-#### `fudge`
```

Comment:
This space is supposed to be there, actually http://spec.commonmark.org/0.24/#atx-headings